### ISSUE TITLE
2D Velocity Damping

### ISSUE THREAD

**Issue by ducharmemp** [rustsim/nphysics#190]
Follow-up of our discussion on discord:

I'm making a top-down 2d tile game, which requires basic collisions and movement mechanics. When a kinematic/dynamic object collides with another dynamic object, since there is no "ground" per se, the objects will travel indefinitely with constant velocity. Applying friction is a pretty simple matter of looping over all of the colliders in the world, but getting the friction right with respect to the material and mass isn't as easy in a way that isn't fragile.

**Comment by frcr** [comment]
Agreed. For a top-down 2d game the loss of velocity is very important. Depending on the type of the game it can be surface friction or aero/hydrodynamic drag.

It is also worth noting that all these types of friction are anisotropic - in the case of a top-down simulated car, the latitudinal friction (dry friction) is much higher than longitudinal (rolling resistance).
